packages/uwt/uwt.0.0.3/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "1.2"
+maintainer: "andreashauptmann@t-online.de"
+authors: [ "andreashauptmann@t-online.de" ]
+license: "LGPL-2.1+ with OCaml linking exception"
+homepage: "https://github.com/fdopen/uwt"
+dev-repo: "https://github.com/fdopen/uwt.git"
+bug-reports: "https://github.com/fdopen/uwt/issues"
+tags: ["clib:libuv"]
+build: [
+  ["omake" "-j%{jobs}%" "lib" "BUILD_LIBUV=true"] {!conf-libuv:installed}
+  ["omake" "-j%{jobs}%" "lib" "BUILD_LIBUV=false"] {conf-libuv:installed}
+]
+install: [
+  ["omake" "install"]
+]
+build-test: [
+  ["omake" "test" ]
+]
+remove: [
+  ["ocamlfind" "remove" "uwt"]
+]
+available: [ocaml-version >= "4.02.1"]
+depends: [
+  "base-unix"
+  "base-bigarray"
+  "base-threads"
+  "base-bytes"
+  "conf-pkg-config" {build}
+  "ocamlfind" {build}
+  "cppo" {build}
+  "omake" {build}
+  "result"
+  "lwt"
+  "ounit" {test & >= "2.0"}
+  "ppx_deriving" {test & >= "2.0"}
+  "ppx_import" {test & >= "1.0"}
+]
+depopts: [
+  "conf-libuv"
+]
+conflicts: [
+ "lwt" {< "2.4.7"}
```

Comment:
Was there a reason for you to move the constraint from `depends` to `conflict` ? If not, the solver better handle the former. Using `confilct` is only required for `depopts`.